app/assets/stylesheets/vendor/svg-arrow.css

```diff
@@ -0,0 +1 @@
+.tippy-box[data-placement^=top]>.tippy-svg-arrow{bottom:0}.tippy-box[data-placement^=top]>.tippy-svg-arrow:after,.tippy-box[data-placement^=top]>.tippy-svg-arrow>svg{top:16px;transform:rotate(180deg)}.tippy-box[data-placement^=bottom]>.tippy-svg-arrow{top:0}.tippy-box[data-placement^=bottom]>.tippy-svg-arrow>svg{bottom:16px}.tippy-box[data-placement^=left]>.tippy-svg-arrow{right:0}.tippy-box[data-placement^=left]>.tippy-svg-arrow:after,.tippy-box[data-placement^=left]>.tippy-svg-arrow>svg{transform:rotate(90deg);top:calc(50% - 3px);left:11px}.tippy-box[data-placement^=right]>.tippy-svg-arrow{left:0}.tippy-box[data-placement^=right]>.tippy-svg-arrow:after,.tippy-box[data-placement^=right]>.tippy-svg-arrow>svg{transform:rotate(-90deg);top:calc(50% - 3px);right:11px}.tippy-svg-arrow{width:16px;height:16px;fill:#333;text-align:initial}.tippy-svg-arrow,.tippy-svg-arrow>svg{position:absolute}
```

Comment:
Should this  (and `tippy.css`) be imported in `stylesheets/common.scss`?

Reply:
I'm not sure why? This is not supposed to be altered.

Reply:
These two CSS files aren't being added anywhere in our stylesheets. I'm assuming we need these styles in the app stylesheets, otherwise there's no need to copy them to `stylesheets/vendor/`.

Reply:
Im sorry maybe I'm misunderstanding, but it's imported here no ?

https://github.com/discourse/discourse/pull/15409/files#diff-e86a39e6d0b163f152db8843fc7d647a3f5d648de4a74f1732bef4474b6e5bc8R5

Reply:
Oh, boy. Yes it is... how did I miss that?